nuxt.config.js

```diff
@@ -1,5 +1,3 @@
-import { defineNuxtConfig } from 'nuxt';
```

Comment:
[`defineNuxtConfig` is now auto-imported](https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.10)